Configurations/mips.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<documentation>
+	<document id="mips32" name="MIPS Architecture Reference Manual" version="A.k">
```

Comment:
If there is a version or revision for this document, could you please add it to `version=""`?